pkg/storage/internalstorage/util_test.go

```diff
@@ -402,15 +402,24 @@ func TestApplyListOptionsToQuery_OrderBy(t *testing.T) {
 			},
 		},
 		{
-			"with not support fields",
+			"order by custom fields asc",
 			[]internal.OrderBy{
-				{Field: "name"},
-				{Field: "group"},
-				{Field: "cluster"},
```

Comment:
We can add a case where there are common fields and custom fields ordered together